iml-graphql-queries/src/log.rs

```diff
@@ -0,0 +1,123 @@
+// Copyright (c) 2020 DDN. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+pub mod logs {
+    use crate::Query;
+    use iml_wire_types::{LogSeverity, MessageClass, SortDir};
+
+    pub static QUERY: &str = r#"
+            query logs($limit: Int, $offset: Int, $dir: SortDir, $message: String, $fqdn: String, $tag: String, $startDatetime: String, $endDatetime: String, $messageClass: [MessageClass!], $severity: LogSeverity) {
+                logs(limit: $limit, offset: $offset, dir: $dir, message: $message, fqdn: $fqdn, tag: $tag, startDatetime: $startDatetime, endDatetime: $endDatetime, messageClass: $messageClass, severity: $severity) {
+                    data {
+                        id
+                        datetime
+                        facility
+                        fqdn
+                        message
+                        message_class: messageClass
+                        severity
+                        tag
+                    }
+                    meta {
+                        total_count: totalCount
+                    }
+                }
+            }
+        "#;
+
+    #[derive(Debug, serde::Serialize)]
+    pub struct Vars {
+        limit: Option<usize>,
+        offset: Option<usize>,
+        dir: Option<SortDir>,
+        message: Option<String>,
+        fqdn: Option<String>,
+        tag: Option<String>,
+        start_datetime: Option<String>,
+        end_datetime: Option<String>,
+        message_class: Option<Vec<MessageClass>>,
+        severity: Option<LogSeverity>,
+    }
+
+    #[derive(Debug)]
+    pub struct Builder {
+        vars: Vars,
+    }
+
+    impl Builder {
+        pub fn new() -> Self {
```

Comment:
Please address clippy issues in this file.

Reply:
I've addressed them in previous update of the PR. If you're still seeing issues, please post them, since my local clippy run doesn't report anything in this file.

Reply:
Do you see them locally if you do:
```
cargo clean && cargo make clippy
```
?
![image](https://user-images.githubusercontent.com/1100657/97179444-37359800-176f-11eb-879b-2c6ee2a96061.png)

Reply:
You should be able to see them by looking at this area of the file in the Github review UI.

Reply:
@johnsonw I only saw them if I close VS Code and do the commands :man_shrugging: 

@jgrund  Done